widgets/Widget - shop.tw

```diff
@@ -33,7 +33,7 @@
 
 						<</link>>
 					<<elseif (_shopItem.sell ?? true)>>
-						
+						(No <<=Item.get(_shopItem.name).name>> to sell) 
```

Comment:
I don't think this is necerssary. I removed this for a reason because it just takes too much space in the shop.

Reply:
Ok. How about a greyed-out sell button when you have nothing to sell?
Better than empty space.

Reply:
Sounds good